src/Microsoft.AspNet.Mvc.Core/ModelBinding/BodyModelBinder.cs

```diff
@@ -66,6 +66,8 @@ public BodyModelBinder()
                     return new ModelBindingResult(modelBindingKey);
                 }
 
+                var valueProviderResult = new ValueProviderResult(model, attemptedValue: null, culture: null);
```

Comment:
- Should `ValueProviderResult` have a one-parameter constructor? Something to perhaps include in this PR.
- Is the current culture correct for these results? They aren't doing conversions.